src/v3/package.json

```diff
@@ -115,21 +115,31 @@
     "jest-junit": "^12.2.0",
     "lint-staged": "^12.2.1",
     "msw": "^1.1.1",
+    "nodemon": "^3.0.1",
     "postcss": "^8.4.13",
+    "postcss-loader": "^7.3.3",
+    "postcss-preset-env": "^8.4.1",
     "postcss-syntax": "^0.36.2",
     "preact-render-to-string": "^5.1.19",
     "preact-svg-loader": "^0.2.1",
     "properties": "^1.2.1",
+    "sass-loader": "^13.2.2",
```

Comment:
why do we need `sass-loader`?

Reply:
Oh, we don't, I'll remove. I had it in there for a while for the playground entry point because I thought we needed to build a *.scss file for the styles on that page but then later realized any styles needed are baked into the SIW styles.